src/App.js

```diff
@@ -1,75 +1,75 @@
-import { connect } from 'react-redux';
-import { lazy, useEffect, Suspense } from 'react';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { connect } from "react-redux";
```

Comment:
So this is what it does? It's basically a linter?

Reply:
basically, as I understand.

Reply:
> So this is what it does? It's basically a linter?

Prettier is for formatting and linters are for catching bugs.

Reply:
I'm going to hold off on this. When the sites almost complete we can circle back to it, if you would like. I already have 2 linters and my editor has multiple bug hunters. You can use what you like but I know what it's like to have to work with unfamiliar tools. I don't want to drive off potential contributors who know the language but not the sintax tools. Does that make since @ShashwatAgrawal20. I can write python programs like a beast but I struggle working on toynet-flask because of things like this. I would rather someone submit some good work and we run this stuff on it then for them not to submit.

Reply:
But wasn't we talking about husky? I apologize, I've been a little under the weather. My brains not working at full compacity the past 2 days.

Reply:
> I'm going to hold off on this. When the sites almost complete we can circle back to it, if you would like. I already have 2 linters and my editor has multiple bug hunters. You can use what you like but I know what it's like to have to work with unfamiliar tools. I don't want to drive off potential contributors who know the language but not the sintax tools. Does that make since @ShashwatAgrawal20. I can write python programs like a beast but I struggle working on toynet-flask because of things like this. I would rather someone submit some good work and we run this stuff on it then for them not to submit.

Yea it's fine, agree to what @Ronnie018 said, btw it's not gonna ask you to format the code, when you have your work done and when you commit the changes to git it automatically gonna format the code so the developer don't have to worry about it and we can keep the code consistent.

Reply:
@ShashwatAgrawal20 I just had to laugh so hard. I've installed Prettier so many times because I've seen a lot of good reviews but I could never figure out what the heck it  actually does and I'd erase it again. I thought all the people were nuts or it was or maybe it was apple thing. I wasn't using git and github thou, I just wrote on my computer and there it stayed. If I needed to back up, I'd use the editors history. 

Oh this is great, I'm still laughing at myself. Okey friend, 2 more questions, are we talking about husky or Prettier  for this P.R. and how do you know it's not going to break stuff after you commit?

Reply:
also you'll need to resolve these merge conflicts.

Reply:
> @ShashwatAgrawal20 I just had to laugh so hard. I've installed Prettier so many times because I've seen a lot of good reviews but I could never figure out what the heck it actually does and I'd erase it again. I thought all the people were nuts or it was an apple thing. I wasn't using git and github thou, I just wrote on my computer and there it stayed. If I needed to back up, I'd use the editors history.
> 
> Oh this is great, I'm still laughing at myself. Okey friend, 2 more questions, are we talking about husky or Prettier for this P.R. and how do you know it's not going to break stuff after you commit?

We are talking about both husky is used to manage the auto formatting or pre commit hook, it doesn't change the code it just prettify It, as like indenting and stuff

Reply:
is this because I'm a girl?

Reply:
You can clone this branch and run `npm i` and try to change some indents and stuff, when you commit this changes its gonna automatically format the code for you

Reply:
> is this because I'm a girl?

Sorry, didn't get it ?

Reply:
prettify, girls are pretty, they like pretty things.

It's fine I was joking with you

Reply:
Will you please update your likes button merge?